### DISCUSSION TITLE
希望出一个插件能关掉b站直播时候的离线检测

### DISCUSSION THREAD

**Discussion by youensChen** [the1812/Bilibili-Evolved#4058] [功能建议 / Ideas]
平时看直播的时候如果一段时间不操作，会自动停掉直播，有没有办法关掉这个东西呢
![image](https://user-images.githubusercontent.com/50787455/224489425-5e80d2d0-a74f-4562-b063-8a3f3e345ca6.png)

**Comment by Geeyun-JY3** [comment]
重复：#1159、#2172

**Reply by youensChen** [reply]
好勒
